UI/BuildingsPanel.cpp

```diff
@@ -107,8 +107,13 @@ void BuildingsPanel::Update() {
         // skip known destroyed and stale info objects
         if (this_client_known_destroyed_objects.contains(object_id))
             continue;
-        if (this_client_stale_object_info.contains(object_id))
-            continue;
+        if (this_client_stale_object_info.contains(object_id)) {
```

Comment:
This seemingly means it won't show any stale buildings on the sidepanel planet panels. At least this needs to be a UI option to enable/disable.

Reply:
As I understand it shows them for the owner of the planet.

Reply:
> As I understand it shows them for the owner of the planet.

because of the logic here or assuming something about how staleness is determined?

Reply:
Because of the logic here. Before my change it skip building indicator for all stale buildings. After my change if option is enabled and the player is the planet's owner we log error and don't skip next code to draw building indicator.